main.py

```diff
@@ -1,10 +1,43 @@
 import init_django_orm  # noqa: F401
+import json
+
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        data = json.load(file)
+
+    for player_name, player_info in data.items():
+
+        race, created = Race.objects.get_or_create(
+            name=player_info["race"]["name"],
+            description=player_info["race"]["description"]
+        )
+
+        if created and player_info["race"].get("skills"):
+            for skill in player_info["race"]["skills"]:
+                Skill.objects.create(
+                    name=skill["name"],
+                    bonus=skill["bonus"],
+                    race=race
+                )
+
+        guild = None
```

Comment:
The `guild` variable is initialized to `None`, which is fine, but it might be clearer to initialize it inside the `if player_info.get("guild"):` block to emphasize its conditional assignment.